pkg/cover/html.go

```diff
@@ -954,8 +986,8 @@ var coverTemplate = template.Must(template.New("").Parse(`
 			<span id="path/{{$dir.Path}}" class="caret hover">
 				{{$dir.Name}}
 				<span class="cover hover">
-					{{if $dir.Covered}}{{$dir.Percent}}%{{else}}---{{end}}
-					<span class="cover-right">of {{$dir.Total}}</span>
+					{{if $dir.Covered}}{{$dir.Percent}}%({{$dir.PercentInCoveredFunc}}%){{else}}---{{end}}
+					<span class="cover-right">of {{$dir.Total}}({{$dir.TotalInCoveredFunc}})</span>
```

Comment:
Lets explain these numbers to the page visitors somehow. Currently we provide no legend.
1. You can add some header line by this commit.
2. It actually could be a good first task for the new people. You can add new github issue and mark it as a "good first task".
3. Your options.

Reply:
Added a missing patch locally https://github.com/google/syzkaller/pull/3144/commits/4143471e4d11d29a009dfdbacdcd83f67a87330b